app/views/api/v1/users/show.json.jbuilder

```diff
@@ -1,4 +1,5 @@
 json.extract! @user, :id, :wechat_id, :username, :profile_image, :first_name, :last_name
 json.goals @user.goals do |goal|
-  json.extract! goal, :id, :amount
+  json.extract! goal, :id, :amount, :nutrient, :unit
```

Comment:
hi @praveen, this is not going to work because:
1. :nutrient just gives you the object, not the nutrient's name
2. goal does not have a :unit, :unit belongs to nutrient, not goal